pkg/util/resizefs/resizefs.go

```diff
@@ -0,0 +1,59 @@
+// +build linux
```

Comment:
The file should be named `resize_linux.go` (I got burned few times with mount.go)